shell/platform/fuchsia/flutter/flatland_external_view_embedder.cc

```diff
@@ -336,6 +344,27 @@ void FlatlandExternalViewEmbedder::SubmitFrame(
       // Reset for the next pass:
       flatland_layer_index++;
     }
+
+    // TODO(fxbug.dev/104956): Setting hit regions for Flatland external view
+    // embedder should match with what is being done in GFX external view
+    // embedder.
+    // Set up the input interceptor at the top of the scene, if applicable.  It
+    // will capture all input, and any unwanted input will be reinjected into
+    // embedded views.
+    if (intercept_all_input_) {
+      FML_CHECK(input_interceptor_node_.has_value());
+      flatland_->flatland()->AddChild(root_transform_id_,
+                                      *input_interceptor_node_);
+
+      // Attach full-screen hit testing shield. Note that since the hit-region
```

Comment:
You can set this once in ctor and forget.

Reply:
Oh, i think i see - create the transform and attach the hit region to it, once. Then add-child this transform to child_transforms_ on each frame submit. Right?

Reply:
Yes that is correct. I initially thought that a transform had to be added as a child before setting its hit region. Turns out I was wrong. Fixed !